src/utils/getBytesOf.spec.js

```diff
@@ -0,0 +1,8 @@
+const { expect } = require('chai');
+const { getBytesOf } = require("./index");
+
+describe('Utils - getBytesOf', function suite() {
+  it('should have getBytesOf return false on unknown type', () => {
```

Comment:
Shall we add a positive test as well?

Reply:
Sure, why not, however, mind that this is a moved file. 
I took note of this, and will do that in a utils improvement PR.

Reply:
Just forget about this then 👍